wallet-new/src/Cardano/Wallet/Kernel/DB/HdWallet.hs

```diff
@@ -211,20 +222,23 @@ eskToHdRootId = HdRootId . InDb . Core.makePubKeyAddressBoot . Core.encToPublic
 -- as a primary key. This however is a slightly larger refactoring we don't
 -- currently have time for.
 newtype HdRootId = HdRootId { getHdRootId :: InDb Core.Address }
-  deriving (Eq, Ord)
+  deriving (Generic, Eq, Ord, Show)
 
 instance Arbitrary HdRootId where
   arbitrary = do
-      (_, esk) <- Core.safeDeterministicKeyGen <$> (BS.pack <$> vectorOf 12 arbitrary)
+      (_, esk) <- Core.safeDeterministicKeyGen <$> (BS.pack <$> vectorOf 32 arbitrary)
```

Comment:
Why exactly :) ?

Reply:
Aeson roundtrips didn't work because of this. 
```
Exception thrown while showing test case:
           Wallet.generate: seed needs to be >= 32 bytes, got : 12
```
(06c8d9424010c5909dd8f8bb8544623b83cd4cd2)

Reply:
:+1: